src/main/java/TaskManager.java

```diff
@@ -0,0 +1,84 @@
+import java.util.ArrayList;
+
+public class TaskManager {
+
+    public static final String ERROR_TASK_NUMBER = "Invalid task number!";
```

Comment:
I like how you assign all the strings as final variables. It makes the code easier to read